### PR TITLE
document setting default device and array creation

### LINKED ISSUE #4220: document setting default device (platform), array creation
Improve our documentation according to [this question](https://github.com/google/jax/issues/957#issuecomment-688939904) and [this answer](https://github.com/google/jax/issues/957#issuecomment-688950524) from @n2cholas on #957.

### PR DESCRIPTION
**Pull request by n2cholas**
Fixes #4220.

### CODE REVIEW

File: docs/faq.rst [hunk 1]
```diff
@@ -67,7 +67,9 @@ and 2) whether it is **committed** to the device or not (the data is sometimes
 referred to as being *sticky* to the device).
 
 By default, JAX arrays are placed uncommitted on the default device
-(``jax.devices()[0]``).
+(``jax.devices()[0]``), which is the first GPU by default. The default
```
Comment (jakevdp) [inline docs/faq.rst:70]: If no GPU is present, `jax.devices()[0]` is the first CPU.

**Review by jakevdp — CHANGES_REQUESTED** [review]
Looks good! One minor nitpick.

File: docs/faq.rst (hunk 1 above)
Comment (n2cholas) [reply docs/faq.rst]: Done!

**Review by jakevdp — APPROVED** [review]
Looks great, thanks!

**Pull request merged**